### PR TITLE
prosody: refactor getting prosody-plugins.

### PR DESCRIPTION
**Pull request by netaskd**
Now we get all files from the deb package jitsi-meet-tokens into /prosody-plugins

### CODE REVIEW

File: prosody/Dockerfile [hunk 1]
```diff
@@ -31,7 +25,18 @@ RUN \
       libssl1.0-dev \
       liblua5.2-dev \
     && apt-cleanup \
-    && rm -rf /etc/prosody
+    && rm -rf /etc/prosody /var/cache/apt
```
Comment (saghul) [inline prosody/Dockerfile]: any reason to duplicate the cleanup steps? we could run them once, at the end

**Comment by netaskd** [comment]
Because every layer is persistent. So for get less image we have to clean unused files in which layer.

File: prosody/Dockerfile (hunk 1 above)
Comment (netaskd) [reply prosody/Dockerfile]: Because every layer is persistent. So for get less image we have to clean unused files in which layer.

Comment (saghul) [reply prosody/Dockerfile]: Ah, true, I always forget.

**Pull request merged**